sniffr/dog_routes/dog_routes.py

```diff
@@ -13,10 +13,19 @@
 def get_dog(dog_id):
     dog_id = int(dog_id)
 
-    queried_dog = db.session.query(Dog).filter_by(dog_id=dog_id).first()
-    queried_dog = process_records(queried_dog)
+    queried_dog = db.session.query(Dog).filter_by(dog_id=dog_id).all()
+    if queried_dog:
+        queried_dog = process_records(queried_dog)
+        payload = jsonify(queried_dog)
+        response = make_response(payload, 200)
+        response.headers["Content-Type"] = "application/json"
+        return response
 
-    return jsonify(queried_dog)
+    else:
+        payload = jsonify({"Message": "Dog Not Found"})
```

Comment:
I'm not sure if jsonify fixes this, but keys are generally lowercase (i.e. "message")

Reply:
It does not do it, so I lowercased them all (I think).